qutebrowser/config/configdata.yml

```diff
@@ -43,6 +43,11 @@ ignore_case:
   default: smart
   desc: Find text on a page case-insensitively.
 
+incremental_search:
+  type: Bool
+  default: False
+  desc: Find text on a page incrementally, renewing the search for each typing character.
+
```

Comment:
Hmm, I think at this point we should rename `ignore_case` to `search.ignore_case`, and call this `search.incremental`. Please do so in a separate commit though - you'll need to keep `ignore_case` with `renamed: search.ignore_case` as value in here, and also adjust any code using `ignore_case`.